shared_model/validators/commands_validator.hpp

```diff
@@ -303,10 +305,52 @@ namespace shared_model {
             answer.addReason(std::move(reason));
           }
         }
+        std::string tx_reason_name = "Trasaction";
+        ReasonsGroupType tx_reason(tx_reason_name, GroupedReasons());
+        validateCreatorAccountId(tx_reason, tx->creatorAccountId());
+        validateCreatedTime(tx_reason, tx->createdTime());
+
+        if (not tx_reason.second.empty()) {
+          answer.addReason(std::move(tx_reason));
+        }
         return answer;
       }
 
      private:
+      void validateCreatorAccountId(
+          ReasonsGroupType &reason,
+          const interface::types::AccountIdType &account_id) const {
+        std::regex e(R"([a-z]{1,9}\@[a-z]{1,9})");
```

Comment:
Check plz if 0 allowed or not.

Reply:
We decided to add this kind of checks in separate PR